src/plasmapy/diagnostics/charged_particle_radiography/synthetic_radiography.py

```diff
@@ -127,22 +165,36 @@ def __init__(
         grids: AbstractGrid | Iterable[AbstractGrid],
         source: u.Quantity[u.m],
         detector: u.Quantity[u.m],
+        dt=None,
+        dt_range=None,
+        field_weighting="volume averaged",
         detector_hdir=None,
+        output_file: Path | None = None,
+        fraction_exited_threshold: float = 0.001,
```

Comment:
The class docstring needs to be updated to include all the kwargs, right now it is missing: 

dt, dtrange, field_weighting, output_file, and fraction_exited_threshold. 

Some of these used to be on `run` so those parts just need to be moved to this docstring.